### PR TITLE
Fixed json parsing when upgrading cucumber 1.0.0 and showing data tables in html report

### PR DESCRIPTION
**Pull request by SirLenz0rlot**
This would fix https://github.com/mavdi/grunt-cucumberjs/issues/71 for me.

I didn't test backwards compatibility, so I upgraded the dependency on cucumber.

### CODE REVIEW

**Comment by SirLenz0rlot** [comment]
Sorry, shouldn't have pushed the fix for #74 on my master, as its part of this PR.

Let me know if there's a problem and/or we need to cherry pick.

**Comment by SirLenz0rlot** [comment]
closing, see #71

File: templates/bootstrap/features.tmpl [hunk 1]
```diff
@@ -53,7 +53,25 @@
...
+                              <% for( var k = 0; k < cells.length; k++ ) { %>
+                                <td>
+                                  <%= cells[k] %>
+                                </td>
+                              <% } %>
+                              </tr>
+                            <% } %>
+                          </table>
+                        <% } %>
+                      </span>
```
Comment (gkushang) [inline templates/bootstrap/features.tmpl]: wow - this was a missing piece. Thanks for adding 👍

File: package.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
     "underscore": "^1.8.3"
   },
   "devDependencies": {
-    "cucumber": "^0.4.8",
+    "cucumber": "^1.0.0",
```
Comment (gkushang) [inline package.json]: 👍

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt-cucumberjs",
   "description": "Generates documentation from Cucumber features",
-  "version": "0.9.2",
+  "version": "0.9.3",
```
Comment (gkushang) [inline package.json]: @SirLenz0rlot Can you un-commit the versioning? We will bump the version with `npm version` to have the commit with versioning.

**Comment by SirLenz0rlot** [comment]
uncommited the version update.

note that the fix for #71 isn't the best solution.

**Pull request merged**